components/api_server/src/routes/report.py

```diff
@@ -178,8 +185,9 @@ def post_report_attribute(report_uuid: ReportId, report_attribute: str, database
 )
 def post_report_issue_tracker_attribute(report_uuid: ReportId, tracker_attribute: str, database: Database):
     """Set the issue tracker attribute."""
-    data_model = latest_datamodel(database)
-    report = latest_report(database, data_model, report_uuid)
+    report = latest_report(database, report_uuid)
+    if report is None:
+        return report_not_found_response(report_uuid)
```

Comment:
Hmm this pattern is so common, is there a nice way to make a kind of `latest_report_or_not_found` wrapper?

Reply:
Ha, yes, that was why I created the `report_not_found_response` helper method. I guess we could make a decorator/plugin that bails if the report is not found. I'll give it a try.